.travis.yml

```diff
@@ -20,25 +20,23 @@ jobs:
     - stage: test
       name: Test and Coverage
       install:
-        # Install Anaconda
+        # Install Micromamba
         # We do this conditionally because it saves us some downloading if the
         # version is the same.
-        - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
-        - bash miniconda.sh -b -p $HOME/miniconda
-        - export PATH="$HOME/miniconda/bin:$PATH"
-        - hash -r
-        - conda config --set always_yes yes --set changeps1 no
-        - conda update -q conda
+        - wget -qO- https://micromamba.snakepit.net/api/micromamba/linux-64/latest | tar -xvj bin/micromamba
+        - ./bin/micromamba shell init -s bash -p ~/micromamba
+        - source ~/.bashrc
+        - micromamba activate base
         # Useful for debugging any issues with conda
-        - conda info -a
-
+        - micromamba info
         # Update python version in environment
         - sed -i -E 's/(python=)(.*)/\1'$TRAVIS_PYTHON_VERSION'/' ./environment.yml
         # Create conda environment
-        - conda env create -n radis-env -f environment.yml
+        - micromamba install xtensor -c conda-forge -y
```

Comment:
@Supriya1702 why was Xtensor added here?

Reply:
@erwanp, In order to install the dependencies of radis mentioned in environment.yml we need to install some prerequisites based in the xtensor library of micromamba, else the dependencies won't install and the build will fail

Reply:
We don't use Xtensor internally in Radis, so it's probably a radis dependency that requires it. Isn't Mamba suppose to find this out by itself?

Reply:
I tried without installing the xtensor, upon creating the virtual environment it was showing errors that some important library missing so can't install dependencies. xtensor contains some important c++ based libraries for GCC compiler:
Following are dependencies in  xtensor:
xtensor == 0.21.5
  ├─ libgcc-ng [>=7.3.0]
  │ ├─ _libgcc_mutex [0.1 conda_forge]
  │ └─ _openmp_mutex [>=4.5]
  │   ├─ _libgcc_mutex already visited
  │   └─ libgomp [>=7.3.0]
  │     └─ _libgcc_mutex already visited
  ├─ libstdcxx-ng [>=7.3.0]
  └─ xtl [>=0.6.9,<0.7]
    ├─ libgcc-ng already visited
    └─ libstdcxx-ng already visited

I believe these libraries are essential for installing basic libraries in micromamba, which can be inferred from the fact that it is very light weight so some dependencies don't come in the base environment.

Also, why has the recent Travis build failed, I tested locally it was working fine. [build][https://app.travis-ci.com/github/radis/radis/jobs/570208518]
File "/home/travis/micromamba/envs/radis-env/bin/pytest", line 8 in <module>
Aborted
The command "xvfb-run pytest --cov=./" exited with 134.

Reference: [https://github.com/mamba-org/mamba]

Reply:
I tried without installing the xtensor, upon creating the virtual environment it was showing errors that some important library missing so can't install dependencies. xtensor contains some important c++ based libraries for GCC compiler:
Following are dependencies in  xtensor:
xtensor == 0.21.5
  ├─ libgcc-ng [>=7.3.0]
  │ ├─ _libgcc_mutex [0.1 conda_forge]
  │ └─ _openmp_mutex [>=4.5]
  │   ├─ _libgcc_mutex already visited
  │   └─ libgomp [>=7.3.0]
  │     └─ _libgcc_mutex already visited
  ├─ libstdcxx-ng [>=7.3.0]
  └─ xtl [>=0.6.9,<0.7]
    ├─ libgcc-ng already visited
    └─ libstdcxx-ng already visited

I believe these libraries are essential for installing basic libraries in micromamba, which can be inferred from the fact that it is very light weight so some dependencies don't come in the base environment.

Also, why has the recent Travis build failed, I tested locally it was working fine. [build](https://app.travis-ci.com/github/radis/radis/jobs/570208518)
File "/home/travis/micromamba/envs/radis-env/bin/pytest", line 8 in <module>
Aborted
The command "xvfb-run pytest --cov=./" exited with 134.
[Reference](https://github.com/mamba-org/mamba)

Reply:
Ok thank you ! 

for local Travis build --> more on Slack !